Cognigy/WebchatWidget
# Bug/71344 start behavior interrupted by privacy screen

This PR prevents the privacy screen from interrupting the 'Auto send message' start behavior.

Success Criteria:
- When 'Auto send message to Bot' is selected in the endpoint settings as a start behavior, the submission of privacy screen should take you to the chat log that will have the get started user message. 
- The get started message should not get collated (i.e. It should have a timestamp and the top margin should be applied)

How to test:
- Create an endpoint and configure the start behavior to 'Auto send message to Bot'.
- Enable privacy screen in endpoint editor
- Clear the local storage and test the bot
- When privacy screen appears, click 'Submit'
- You should now see the chat log with the Get started message including a timestamp.
